task.go

```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/mitchellh/mapstructure"
 )
 
+// Not sure where best to put this?
```

Comment:
Hmm, maybe we can just make a new `responses.go` file for these. It looks like there are a couple others that could be moved there too, though that's not necessary as part of your PR (`configserver.SaveConfigResponse` and `atc.ConfigResponse`).

Reply:
Sounds good.  I've only moved the `ClearTaskCacheResponse` struct to the `responses.go`, but might move the other responses in a separate PR.